tilt_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: tilt_joint
    joint_speed: 0.05
    motor:
        id: 1
        init: 0
        min: 0
        max: 4096
